pkg/execution/svccatadmission/internaldns.go

```diff
@@ -9,7 +9,7 @@ import (
 
```

Comment:
rename the file to `platformdns.go`?

Reply:
I'll deal with webhooks reanming in a separate PR